package.json

```diff
@@ -1,16 +1,18 @@
 {
   "name": "@ssense/vue-carousel",
-  "version": "0.3.3",
+  "version": "0.4.0",
   "description": "A flexible, responsive, touch-friendly carousel for Vue.js",
   "main": "src/index.js",
   "scripts": {
     "build": "webpack --config config/webpack.min.conf.js && npm run updateDocAssets",
     "updateDocAssets": "cp ./dist/vue-carousel.min.js ./docs/themes/vue/source/js/",
-    "test": "npm run test:unit",
-    "test:unit": "./node_modules/.bin/karma start test/unit/karma.conf.js --single-run",
+    "test": "npm run test:client --watch",
+    "test-coverage": "npm run build && npm run test:client:coverage",
+    "test:client": "jest --config tests/client.jest.json --forceExit",
+    "test:client:coverage": "jest --config tests/client.jest.json --coverage --forceExit",
```

Comment:
May I know what's the reason behind the `--forceExit` options? I'm trying to work on increasing the code coverage.

Reply:
Hey @josephting, this flag tells just to exit once  the test run is finished, in case for some reason. You can read more about it [here](https://jestjs.io/docs/en/cli.html#forceexit).

Feel free to make modifications to this if you don't think it's necessary anymore, the testing scripts haven't been audited in a long time

Reply:
Got it, thanks! Just curious why was it needed in the first place but I'll remove it as I see fit. Aiming to get a PR ready next week.